jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -180,9 +226,14 @@ public FromConfiguration() {
 
     @Nullable @Parameter private String image;
     @Parameter private List<String> tags = Collections.emptyList();
-    @Nullable @Parameter private String credHelper;
+    @Parameter private CredHelperParameters credHelper;
```

Comment:
I think we can just do
```suggestion
    @Parameter private CredHelperParameters credHelper = new CredHelperParameters();
```
and remove the constructor you added.

Reply:
Got it